h/schemas/api/group.py

```diff
@@ -1,12 +1,14 @@
 """Schema for validating API group resources."""
 
+import colander
+
 from h.i18n import TranslationString as _
 from h.models.group import (
     GROUP_DESCRIPTION_MAX_LENGTH,
     GROUP_NAME_MAX_LENGTH,
     GROUP_NAME_MIN_LENGTH,
 )
-from h.schemas.api.moderation import ModerationStatusMixin
```

Comment:
I tried mixins in the [previous PR](https://github.com/hypothesis/h/pull/9505) (still a draft) but while the docs do go into detail aobut heiriahnce I found it tricky to make it work.

Reply:
Yeah. I think the reuse-by-composition approach is probably better